src/main/java/underdogs/devbie/notice/domain/NoticeDescription.java

```diff
@@ -0,0 +1,51 @@
+package underdogs.devbie.notice.domain;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Embeddable;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import underdogs.devbie.notice.expception.CreateFailException;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Embeddable
+@Getter
+@EqualsAndHashCode
+public class NoticeDescription {
+
+    @Enumerated(value = EnumType.STRING)
+    @ElementCollection
+    @CollectionTable(joinColumns = @JoinColumn(name = "notice_id"), name = "language")
+    private Set<Language> languages;
+
+    @Lob
+    @Column(columnDefinition = "CLOB")
+    private String content;
+
+    public NoticeDescription(Set<String> languages, String content) {
+        validateParameters(languages, content);
+        this.languages = languages.stream().map(Language::from).collect(Collectors.toSet());
```

Comment:
```suggestion
        this.languages = languages.stream()
            .map(Language::from)
            .collect(Collectors.toSet());
```
한분께서 한줄에 줄여쓰는걸 좋아하시나보군요!
첫주에 컨벤션으로 정했듯 가독성을 위해 .을 기준으로 줄바꿈을 해주는게 어떨까요?